src/patterns/filtering/single-selection.stories.ts

```diff
@@ -0,0 +1,97 @@
+import { Component } from "@angular/core";
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs } from "@storybook/addon-knobs/angular";
+import { TableModule, TableModel, TableHeaderItem, TableItem } from "../../table/table.module";
+import { DropdownModule } from "../../dropdown/dropdown.module";
+
+@Component({
+    selector: "app-sample-single-selection",
+    template: `
+    <ibm-table-container>
+        <p ibmText style="float: left; margin-right: 10px; line-height: 30px">Filter by: </p>
```

Comment:
Same comment as the multiselection table example ... there's a better way to do this ... the carbon grid can help out, and some of the directives in the table module may help as well